charts/gardener-extension-csi-driver-lvm/values.yaml

```diff
@@ -0,0 +1,44 @@
+image:
+  repository: ghcr.io/metal-stack/gardener-extension-csi-driver-lvm
+  tag: latest
+  pullPolicy: IfNotPresent
+
+replicaCount: 1
+resources: {}
+
+controllers:
+  concurrentSyncs: 5
+  healthcheck:
+    concurrentSyncs: 5
+  heartbeat:
+    renewIntervalSeconds: 30
+  ignoreOperationAnnotation: false
+
+disableControllers: []
+
+# imageVectorOverwrite: |
+#   images:
+#   - name: pause-container
+#     sourceRepository: github.com/kubernetes/kubernetes/blob/master/build/pause/Dockerfile
+#     repository: gcr.io/google_containers/pause-amd64
+#     tag: "3.0"
+#     version: 1.11.x
+#   - name: pause-container
+#     sourceRepository: github.com/kubernetes/kubernetes/blob/master/build/pause/Dockerfile
+#     repository: gcr.io/google_containers/pause-amd64
+#     tag: "3.1"
+#     version: ">= 1.12"
+#   ...
+
+config:
+  clientConnection:
+    acceptContentTypes: application/json
+    contentType: application/json
+    qps: 100
+    burst: 130
+
+  devicePattern: /dev/loop10[0,1]
```

Comment:
Maybe it makes sense to use the same default value here as we did in the csi-lvm deployment: https://github.com/metal-stack/gardener-extension-provider-metal/blob/bfa57a27e6a1b5941fdea07e1945b2bd75a45c9e/charts/internal/shoot-storageclasses/templates/storageclasses.yaml#L130-L132